CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2020 Ultimaker B.V.
+# pynest2d is released under the terms of the LGPLv3 or higher.
+
+project(pynest2d)
+cmake_minimum_required(VERSION 3.6)  # Lowest version it's been tested with.
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
+
+# Requirements.
+find_package(PythonInterp 3.5 REQUIRED)  # Dependency of SIP.
```

Comment:
add a `# FIXME: Use the new FindPython3 module rather than these. New in CMake 3.12.`

Reply:
FindPython3 finds the most modern version of Python that matches a certain minimum version. FindPythonInterp matches on a specific version.

On our CI server running CentOS 7, FindPython3 finds the most modern version of Python installed, which is then CPython 3.6 from CentOS itself, instead of the CPython 3.5 which is installed through cura-build-environment. So either we must remove the installation of CPython 3.6 from that server, or we cannot use that `FindPython3`.

Or we must upgrade to a newer version of Python, of course, or downgrade our CI server.